react/src/scss/variables.scss

```diff
@@ -0,0 +1,7 @@
+// Use this file to keep track of variables and colors we are using
+// makes it easier to change things later
+
+:root{
+    --very-light-grey: #F4F4F4;
+    --light-grey: #EEEEEE;
```

Comment:
https://sass-lang.com/guide#topic-2
This a link to how you can define variables in sass.
Also, I believe `lightgray` (or `lightgrey`) is already a colour

Reply:
oh your right